suite-native/alerts/src/components/AlertSheet.tsx

```diff
@@ -21,6 +22,9 @@ type AlertSheetProps = {
     alert: Alert;
 };
 
+const screenWidth = getScreenWidth();
+const screenHeight = getScreenHeight();
```

Comment:
I'd go with the same naming as in other files:
```suggestion
const SCREEN_WIDTH = getScreenWidth();
const SCREEN_HEIGHT = getScreenHeight();
```